enterprise/migrations/0091_auto_20200311_0808.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
Should we rename this migration file to reflect its purpose, rather than use the auto-generated name?